src/views/HomePage.vue

```diff
@@ -0,0 +1,303 @@
+        <Button 
+            @click="setSortBy('releaseDate')" 
+            title="RELEASE DATE" 
+            :type="sortBy === 'releaseDate' ? 'primary' : 'secondary'" 
+        />
+        <Button 
+            @click="setSortBy('rating')" 
+            title="RATING" 
+            :type="sortBy === 'rating' ? 'primary' : 'secondary'" 
+        />
+      </div>
+    </section>
+
+    <section class="main">
+        <div v-if="movies.length > 0" class="container" >
+            <div class="movie-card" v-for="movie in movies" :key="movie.id">
+                <div class="hover-cover">
+                    <div class="rating-container">
+                        <div class="rating-circle">
+                        <span class="rating-number">{{ movie.rating }} / 5</span>                      
+                        </div>
+                    </div>
+                    <div class="rating-description">{{ movie.description }}</div>
+                </div>
+                <img v-lazyload="movie.imagePath" :alt="movie.title" />
+
+                <div class="movie-info">
+                    <div class="movie-title">
+                        <h3>{{ movie.title }}</h3>
+                        <p>{{ movie.genre }}</p>
+                    </div>
+
+                    <div class="movie-release-date">
+                        {{ movie.releaseDate }}
+                    </div>
+                </div>
+            </div>             
+        </div>
+        <div v-else class="warning-info">
+            <h2>No movies found</h2>
+        </div>   
+    </section>
+
+    <footer>
+        <LogoText />
+    </footer>
+
+</template>
+
+<style lang="scss" scoped>
+$primary-color: #F65261;
+$secondary-color: #555555;
+
+$movie-card-width: 300px;
+$movie-card-height: 450px;
+
+header {
+    width: 100%;
+    height: 17rem;
+    color: white;
+
+    img {
+        position: absolute;
+        z-index: -1;
+        width: 100%;
+        height: 100%;
+        filter: brightness(0.2);
+    }
+
+    .container {
+        .logo-placeholder {
+            padding: 2rem;
+        }
+
+        .search {
+            display: flex;
+            align-items: center;
+            flex-flow: column;
+
+            div {
+                width: 50%;
+            }
+
+            .search-field {
+                display: flex;
+                margin-bottom: 1rem;
+            }
+        }
+    }
+}
+
+.nav-bar {
+    display: flex;
+    justify-content: space-evenly;
+    background-color: $secondary-color;
+    padding: 1rem 0 1rem 0;
+    color: white;
+
+    .body-sort {
+        display: flex;
+        align-items: center;
+    }
+
+    p {
+        font-weight: bold;
+    }
+}
+
+.main {
+    background-color: #232323;
+    padding-left: 1.5rem;
+
+    .warning-info {
+        height: 546px;
+        color: white;
+        font-size: x-large;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .container {
```

Comment:
Read about this: https://getbem.com/introduction/ and https://andrew-barnes.medium.com/bem-and-sass-a-perfect-match-5e48d9bc3894
This will help you write cleaner css code.